### DISCUSSION TITLE
how can we disable certain INFO logs on api call ?

### DISCUSSION THREAD

**Discussion by neonarc4** [fastapi/fastapi#11619] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
@app.get("/stuff")
 return "panda"
```

### Description

i want to remove its INFO log which generate in command panel

### Operating System

Windows

### FastAPI Version

0.109.0

### Pydantic Version

2.6.0

### Python Version

3.10

### Additional Context

![image](https://github.com/tiangolo/fastapi/assets/140290288/b6a706f9-3394-4cc4-a183-88859a714cf0)
i just want to hide these on command panel it keep spam

**Comment by Kludex** [comment, marked as answer]
`uvicorn --log-level warning` or something like that...
